src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,94 @@
+package homework_2.random_chars_table;
+
+import homework_2.random_chars_table.exception.RandomCharsTableException;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static homework_2.random_chars_table.utils.MessageType.ERROR_MESSAGE;
+import static homework_2.random_chars_table.utils.MessageType.INFO_MESSAGE_FOR_COLUMNS;
+import static homework_2.random_chars_table.utils.MessageType.INFO_MESSAGE_FOR_ROWS;
+import static homework_2.random_chars_table.utils.MessageType.INFO_MESSAGE_FOR_STRATEGY;
+import static homework_2.random_chars_table.utils.MessageType.INTEGER_ERROR_MESSAGE;
+import static homework_2.random_chars_table.utils.MessageType.STRATEGY_ERROR_MESSAGE;
+import static homework_2.random_chars_table.utils.Utils.printMessage;
+import static java.lang.System.lineSeparator;
+
+public class RandomCharsTable {
+    private static final int MAX_CHAR = 90;
+    private static final int MIN_CHAR = 65;
+
+    public void start() {
```

Comment:
Please refactor to insert 3 3 even in one line

Reply:
refactored